src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -20,6 +20,7 @@
         "ext-xml": "*",
         "symfony/cache": "^4.4|^5.0",
         "symfony/config": "^4.2|^5.0",
+        "symfony/contracts": "~1.0",
```

Comment:
service-contracts (not needed if we add it to http-kernel 4.4, but we're maybe not, per another comment of mine :))